### ISSUE TITLE
OsGeo4W V2 update

### ISSUE THREAD

**Issue by nicogodet** [haubourg/custom-osgeo4w-qgis#2]
I [started working](https://github.com/haubourg/custom-osgeo4w-qgis/compare/haubourg:main...nicogodet:isl?expand=1) on the update of your awesome work. 

Few typos and some changes in postinstall and preremove script (avoid the use of nircmd).
Still WIP but I will isolate the main change and propose a PR, I hope we can both work on this.

**Comment by haubourg** [comment]
Wow that's awesome, thanks a lot @nicogodet !

A Pull request will be very welcome when you feel ready. 
At first glance, you will have to replace all 'isl' occurences back to 'custom' to keep it generic. 
I'll probably work on this for my new employer again, most probably in 2022, but I'll be happy to test. 
Also feel free to talk about this on qgis-developper  and osgeo4W lists and bug trackers if you find upstream issues or enhancement.  

poke @Utopiageo this can help you upgrading faster to next OSGEO4W installer generation

**Comment by nicogodet** [comment]
Of course, the `isl` branche will not be used for PR. I will create another branch and pick some fixes from isl.

Currently, install works like a charm with V2, I still have issue with desktop shortcut. `OSGEO4W_DESKTOP_LINKS` equals `0` even if I don't specify `--no-desktop`.
I need to figure out how to create a oneline command for upgrades.
